summary_workflows/quantification/benchmarking_dockers/README.md

```diff
@@ -1,6 +1,6 @@
-# TCGA benchmarking Docker declarations
+# APAeval benchmarking Docker declarations
 
-OpenEBench TCGA benchmarking Docker declarations, which define the architecture of benchmarking workflows to be implemented in OpenEBench.
+OpenEBench/iRNA COSI APAeval benchmarking Docker declarations, which define the architecture of benchmarking workflows to be implemented in OpenEBench.
 
 **NOTE for developers.** In order to make the workflow containers reproducible and stable in the long-term, make sure to use specific versions in the container base image (e.g.*ubuntu:16.04*, NOT *ubuntu:latest*).
 
```

Comment:
In the following section, links have to be adapted to apaeval (are still tcga) (I can't directly comment within the section for whatever reason, but I see 5 occurrences of tcga)